unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs

```diff
@@ -266,6 +266,9 @@ public void SetAgentRadius(float agentRadius = 2.0f)
         public GameObject FindObjectInVisibleSimObjPhysics(string objectID) {
             GameObject target = null;
 
+            if (VisibleSimObjPhysics is null) {
```

Comment:
We should just call VisibleSimObjs (i can refactor this when it gets merged to main on the IsInteractableBranch) instead of calling LateUpdate().  LateUpdate is assumed to be called only once per frame by the Unity Lifecycle and if for some reason VisibleSimObjs were remove from LateUpdate, this would not work.  As well, VisibleSimObjs is only populate for the Editor and WEBGL.

Reply:
Fixed, I ended up refactoring this function a bit (and as you note, I think this function can be completely replaced once your IsInteractableBranch fixes are in).